src/main/java/com/facebook/openwifirrm/modules/ConfigManager.java

```diff
@@ -63,8 +65,14 @@ private class DeviceData {
 	/** Is the main thread sleeping? */
 	private final AtomicBoolean sleepingFlag = new AtomicBoolean(false);
 
-	/** Was a manual config update requested? */
-	private final AtomicBoolean eventFlag = new AtomicBoolean(false);
+	/**
+	 * Set of venues for which manual config updates have been requested.
+	 * This is a thread-safe (concurrent, not synchronized) set, since it is
+	 * backed by a {@code ConcurrentHashMap}, but is still statically typed as
+	 * {@code Set<String>} since there is nothing like ConcurrentHashSet in
+	 * Java.
```

Comment:
you do not need to explain what `ConcurrentHashMap` is in this comment, readers can look it up themselves

Reply:
I'm not exactly using ConcurrentHashMap (I'm calling `newKeySet()` which returns a thread-safe set). Do you think I should remove most of this javadoc?